base/commands/config/config_import.go

```diff
@@ -23,16 +23,16 @@ Currently importing only Viridian connection configuration is supported.
 	
 1. On Viridian console, visit:
 	
-	Dashboard -> Connect Client -> Quick connection guide -> Go
```

Comment:
We should also update the steps in documentation file `clc-config.adoc`

Reply:
Good catch, updated at: https://github.com/hazelcast/hazelcast-commandline-client/pull/312/commits/f226f9ee82e278e03905c159df92ad9e88f8ed45